tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Gateway_Test.php

```
@@ -505,6 +505,66 @@ public function test_payment_fields_outputs_fields() {
+		$gateway->method( 'get_return_url' )->willReturn( self::MOCK_RETURN_URL );
+		$gateway->enabled = 'yes';
+
+		// Make is_checkout() return true so payment_scripts() passes its page guard.
+		add_filter( 'woocommerce_is_checkout', '__return_true' );
+
+		wp_deregister_script( 'wc-stripe-upe-classic' );
+
+		$gateway->payment_scripts();
+
+		// The script should be registered with the version we derived above.
+		$script_is_registered = wp_script_is( 'wc-stripe-upe-classic', 'registered' );
```

PHPMD (2.15.0)

[error] The variable $script_is_registered is not named in camelCase. (undefined)

(CamelCaseVariableName)